### DISCUSSION TITLE
Phosphor icons angular library

### DISCUSSION THREAD

**Discussion by mmed199** [phosphor-icons/homepage#445] [Q&A]
Hello, is there any maintained phosphor-icons Angular Library? I use your the JS library directly in my angular apps and I want to create a library to easly integrate the icons.
I found this : https://www.npmjs.com/package/ngx-phosphor-icons
But I don't think it's maintained

**Comment by rektdeckard** [comment]
Hi @mmed199! There's no active port that I know of, sorry!
